tests/5.0/metadirective/test_metadirective_arch_is_nvidia.F90

```diff
@@ -0,0 +1,61 @@
+!===--- test_metadirective_arch_is_nvidia.F90 -------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! Test for metadirectives based on OpenMP 5.0 examples metadirective.1-3.c
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_metadirective_arch_is_nvidia
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_VERBOSE(test_metadirective1() .NE. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION test_metadirective1()
+    INTEGER,DIMENSION(N):: v1, v2, v3
+    INTEGER:: target_device_num, host_device_num, default_device
+    INTEGER:: errors, i, j
+
+    errors = 0
+
+    DO i = 1, N
+       v1(i) = i+1
+       v2(i) = -(i+1)
+    END DO
+
+    host_device_num = omp_get_initial_device()
+    
+    default_device = omp_get_default_device()
+
+    !$omp target map(to:v1,v2) map(from:v3,target_device_num) device(default_device)
+    !$omp begin metadirecitve when(device={arch("nvptx")}: teams distribute parallel do) default(parallel do)
+    DO i = 1, N
+    !$omp atomic write
+       target_device_num = omp_get_device_num()
+    !$omp end atomic
+       v3(i) = v1(i) * v2(2)
+    END DO
+    !$omp end metadirective
+    !$omp end target teams
```

Comment:
```suggestion
    !$omp end target
```
`!$omp end target` was missing – additionally, the "end teams" is already implied by `end metadirective`.

_(Regarding the latter, OpenMP has (here quoting 5.2): "For any directive variant that is selected to replace the **begin metadirective** directive, the **end metadirective** directive is implicitly replaced by its paired **end** directive to demarcate the statements that are affected by or are associated with the directive variant [...]")_